### ISSUE TITLE
Improve accessibility of admin-ui

### ISSUE THREAD

**Issue by Evodam** [keycloak/keycloak#36480]
### Description

As far as I can see account-ui and admin-ui lack several accessibility features like:

- use <p> for paragraphs instead of div
- assign error message to their respective form field with aria attributes
- group form fields and labels with <fieldset>
- eye-icon for password hints cannot be focused

### Discussion

https://github.com/keycloak/keycloak/discussions/36474
